FORMULA

```diff
@@ -1,8 +1,9 @@
 name: nginx
-os: RedHat, Debian, Ubuntu, Suse, Gentoo
-os_family: RedHat, Debian, Suse, Gentoo
-version: 201507
+os: Debian, Ubuntu, Raspbian, RedHat, Fedora, CentOS, Suse, openSUSE, Gentoo, Funtoo, Arch, Manjaro, Alpine, FreeBSD, OpenBSD, Solaris, SmartOS, Windows, MacOS
+os_family: Debian, RedHat, Suse, Gentoo, Arch, Alpine, FreeBSD, OpenBSD, Solaris, Windows, MacOS
```

Comment:
Both `os` and `os_family` in the `template-formula` were given as a full list that can be selected from.  So best to only capture the platforms which definitely work with this formula.